spec/features/account_manager/duplicating_an_entry_spec.rb

```diff
@@ -1,15 +1,14 @@
 require 'spec_helper'
 
-feature 'account manager duplicating an entry' do
-  scenario 'from own account' do
-    account = create(:account)
-    entry = create(:entry, :title => 'Test Entry', :account => account)
+feature 'duplicating an entry' do
+  scenario 'as publisher of entry' do
+    user = Dom::Admin::Page.sign_in_as(:editor)
```

Comment:
The way to create an unprivileged user is now to say sign_in_as(:editor), without passing a pair with key 'on' in the options hash. I just adjusted it so that this should now work exactly as it did before. sign_in_as(:account_manager) also works as before, however sign_in_as(:account_manager, account: account) will soon not work anymore because the account Membership replaces the account attribute. It has to be replaced by sign_in_as(:account_manager, on: account) or even finally by sign_in_as(:manager, on: account) if the role attribute on user can be let alone.